validation/postValidation.ts

```diff
@@ -0,0 +1,16 @@
+import * as yup from "yup";
+
+export const validateUserId = {
+    userId: yup.string().trim().required(),
```

Comment:
Great work. We don't need to validate the userId for posts as we get it directly from session data. For future reference Yup has a uuid checker which will make sure it is actually a uuid rather than just a string: https://github.com/jquense/yup#stringuuidmessage-string--function-schema

Reply:
Hi Jet,
I want to push this pull request only for user validation. I will pull request for post validation next time. Can I merge it?

Reply:
I changed the code to check if it is uuid or not. I will create another pull request for post validation after fix validatePost.

Reply:
Ok, that's fine. Let's merge once the tests all pass. I suggest that you create a new branch to keep the changes separate in future, though 😉